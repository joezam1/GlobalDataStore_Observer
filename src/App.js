import React from 'react';


import TopSection from './TopSection.js';
import MidSection from './MidSection.js';
import BottomSection from './BottomSection.js';


import Counter from './Counter.js';

export default function App(){
    return( <div>
            <Counter/>
            <TopSection/>
            <MidSection/>
            <BottomSection/>
        </div>
    )
}