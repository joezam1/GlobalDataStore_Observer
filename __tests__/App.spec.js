import React from 'react';
import {render} from '@testing-library/react';
import App from '../src/App.js';


describe('file App.js', function(){
    test('it should take a snapshot', function(){
        var container = render(<App/>);
        expect(container.asFragment(<App/>)).toMatchSnapshot();
    });
});