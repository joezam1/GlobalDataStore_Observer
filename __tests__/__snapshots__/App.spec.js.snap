// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file App.js it should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="section-counter"
    >
      <h1
        data-testid="value"
      >
        count: 99
      </h1>
      <button
        data-testid="button-increment"
        type="button"
      >
        Increment
      </button>
    </div>
    <div
      class="section-top"
    >
      top section
      <br />
      Value:
      <span
        data-testid="value"
      />
      <br />
      <button
        data-testid="button-increment"
        type="button"
      >
        Increment
      </button>
    </div>
    <div
      class="section-mid"
      data-testid="mid-section"
    >
      Middle section
      <span
        data-testid="value"
      />
      <br />
    </div>
    <div
      class="section-bottom"
    >
      "this is bottom section"
      <br />
      Value: 
      <button
        data-testid="button-decrement"
        type="button"
      >
        Decrement
      </button>
    </div>
  </div>
</DocumentFragment>
`;
